add-on/content/SerpMonitor.jsm

```diff
@@ -165,9 +167,23 @@ this.SerpMonitor = {
     };
 
     log.info(`Reporting to Telemetry: ${type} with additional info`, additionalInfo);
+    let fullEnvironment = TelemetryEnvironment.currentEnvironment;
+    let defaultSearchEngine =
+      fullEnvironment.settings ? fullEnvironment.settings.defaultSearchEngine : "";
+    let activeExperiment =
+      fullEnvironment.addons ? fullEnvironment.addons.activeExperiment : "";
```

Comment:
Are you sure we only need the active experiment and not the experiments array here? I guess we need to ask metrics folks to clarify.